test/integration/etcd_storage_path_test.go

```diff
@@ -540,6 +576,14 @@ var etcdStorageData = map[schema.GroupVersionResource]struct {
 	},
 	// --
 
+	//// k8s.io/kubernetes/pkg/apis/events/v1
```

Comment:
what's the issue here?

Reply:
> what's the issue here?

Validation doesn't want to accept it.  Events are transient and best effort, so a skip is tolerable for a few weeks and if we lose them all, we won't really care.

Reply:
add follow up